Exercise_Stringers.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import "fmt"
+
+type IPAddr [4]byte
+
+// Byte2String convert Byte type to String type
+func Byte2String(x byte) string {
+	var s string
+	for x/10 != 0 {
+		r := x % 10
+		x = x / 10
+		s = s + string(r+48)
+	}
+	s = s + string(x+48)
+	var res string
+	for i := len(s) - 1; i >= 0; i-- {
+		res = res + string(s[i])
+	}
+	return res
+}
+
+// TODO: Add a "String() string" method to IPAddr.
+
+func (ipaddr IPAddr) String() string {
```

Comment:
sao em làm cái j mà nó phức tạp thế này. bài này đơn giản là dùng Stringer để  show  IPAddr{1, 2, 3, 4} should print as "1.2.3.4".